docs/install/airgap.md

```diff
@@ -40,3 +44,34 @@ system-default-registry: "registry.example.com:5000"
 ```
 
 **Note:** The `system-default-registry` parameter must specify only valid RFC 3986 URI authorities, i.e. a host and optional port.
+
+### RKE2 Install.sh Script Install
+
+You may use the `install.sh script` and define the `INSTALL_RKE2_ARTIFACT_PATH` to reduce steps and create the systemd service file.
```

Comment:
```suggestion
`install.sh` may be used in an offline mode by setting the `INSTALL_RKE2_ARTIFACT_PATH` variable to a path containing pre-downloaded artifacts. This will run though a normal install, including creating systemd units.
```